conf/defaults.ini

```diff
@@ -57,6 +57,17 @@ enable_gzip = false
 cert_file =
 cert_key =
 
+# TLS settings.
+# The minimum supported version of TLS. Either "tls10", "tls11" or "tls12", default is "tls10".
+tls_min_version = tls10
```

Comment:
Prior to this change tls10 was the default, after it's merged you can change it to tls12 as the default, which I also recommend, but may not work for everyone.